.github/workflows/ssh-runner.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on: [single-gpu, nvidia-gpu, "${{ github.event.inputs.runner_type }}", ci]
     container:
       image: ${{ github.event.inputs.docker_image }}
-      options: --gpus all --privileged --ipc host -v /mnt/cache/.cache/huggingface:/mnt/cache/
+      options: --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface/diffusers:/mnt/cache/ --gpus 0 --privileged
```

Comment:
Why not `--gpus all`?

Reply:
These were all meant to run on single gpu machines, so there's only one GPU available anyway.

Reply:
Disregard sorry.